posthog/hogql/database/schema/test/__snapshots__/test_revenue_analytics.ambr

```diff
@@ -970,8 +970,82 @@
         FROM
           (SELECT count() AS total,
                   toStartOfDay(toTimeZone(e.timestamp, 'UTC')) AS day_start,
-                  ifNull(nullIf(toString(replaceRegexpAll(nullIf(nullIf(JSONExtractRaw(e.person_properties, '$virt_revenue'), ''), 'null'), '^"|"$', '')), ''), '$$_posthog_breakdown_null_$$') AS breakdown_value_1
+                  ifNull(nullIf(toString(e__pdi__revenue_analytics.revenue), ''), '$$_posthog_breakdown_null_$$') AS breakdown_value_1
```

Comment:
This query is crazy now 🤯 

It's very cool! I'm assuming you are keeping an eye on perf 😉